packages/yoga/src/Progress/native/Progress.jsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import styled from 'styled-components';
 import { string, number, shape, oneOfType, oneOf } from 'prop-types';
+import {
+  backgroundColor,
+  border,
+  elevation,
+  positions,
+  display,
+  spacing as Spacing,
```

Comment:
```suggestion
  spacing as systemSpacing,
```

just to not confuse JSX parser